packages/driver/cypress/integration/commands/querying/root_spec.js

```diff
@@ -35,6 +35,23 @@ describe('src/cy/commands/querying', () => {
       cy.root().should('have.class', 'foo').and('have.class', 'bar')
     })
 
+    // https://github.com/cypress-io/cypress/issues/19985
```

Comment:
Can we use a timeout of 150ms here (and setTimeout of 50ms), rather than 4200 + 4300?

Reply:
I rewrote the test like below:

```js
cy.on('fail', (err) => {
  expect(err.message).to.not.contain('4000ms')
  done()
})

cy.root({ timeout: 50 }).should('contain', 'root world')
```

I wrote the first test like that because the default timeout is 4000ms, so I needed to test that `cy.root` can wait for more than 4000ms. If it's shorter than that, we cannot be sure that `cy.root({ timeout: 150})` worked because it's fixed or 150ms is simply lower than 4000ms. 

But I realized that I can test it faster like the new code.